webpack.config.js

```diff
@@ -65,11 +65,27 @@ const webpackConfig = {
 			{
 				include: /node_modules\/tinymce/,
 				loader: 'imports?this=>window',
-			}
+			},
+			{
+				test: /\.scss$/,
+				loaders: [
+					'isomorphic-style-loader',
+					'css-loader?camelCase=dashes&importLoaders=1',
+					'sass-loader'
+				]
+			},
+			{
+				test: /\/client\/components\/vertical-nav.+index\.jsx$/,
```

Comment:
* sass imports are missing in *.sass files (variables, mixins, functions)
* we need to find a way to conditionally load style.scss - don't try to load the file if it is missing
* we need to cover the case where we override index.jsx with a different name using `package.json`
* everything else that is going to pop up :)